### DISCUSSION TITLE
Training procedure

### DISCUSSION THREAD

**Discussion by jregio** [RosettaCommons/RFdiffusion#239] [Q&A]
Hello, I would like to know where I can find the code for training the diffusion model.  Is this available somewhere in the repository?  Thank you!

**Comment by roccomoretti** [comment]
The training code is currently unreleased. I am personally unaware if there's plans or a timeframe for releasing it.

If you're interested in details about how things were trained, please see the [original paper](https://www.nature.com/articles/s41586-023-06415-8), particularly the associated Supplementary Information file, which contains many of the details. Another thing to take a look at is the lectures about RFdiffusion which are available on YouTube. There are several talks - particularly by Joe Watson, David Juergens and Sergey Ovchinnikov - which may provide additional insights on training. (David Baker's talks are usually for more general audiences, and typically do not go into the technical ML details.)
